public/styles/global.css

```diff
@@ -15,9 +15,13 @@ header {
 header .board-title {
 	float: right;
 	text-transform: uppercase;
-	font-size: 48px;
+	font-size: 32px;
```

Comment:
Had to bring this down because the larger text was pushing the rest of the page down too far. Alternatively I could try moving the whole block up, but I'm not sure what you prefer.

Reply:
I've removed these changes.